packages/vertexai/src/models/generative-model.test.ts

```diff
@@ -165,14 +165,33 @@ describe('GenerativeModel', () => {
     );
     restore();
   });
+  it('overrides base model params with startChatParams', () => {
```

Comment:
Is this still needed after adding this case to the line 252 test?

Reply:
nope, removed.